### ISSUE TITLE
Gene names and HGNC symbols

### ISSUE THREAD

**Issue by MitchellFlagg** [ucsd-ccbb/ARMORsmith#23]
TLDR: Some ARMOR data is reported with gene names that are not the HGNC standard symbols, how to understand/address this?

So I noticed this when trying to take external lists of genes for each cancer test and use them to determine the test space and to differentiate pertinent negatives from untested genes - when trying to line up that external gene list against the unique genes for which findings were reported in the ARMOR data, I noticed some odd cases where there were reported results for genes whose symbols weren't in that external list. I looked into some of these genes individually by name and confirmed for many of them that the names used in the ARMOR data were not their official HGNC gene symbols.

I wanted to find out what I might be missing about the reasons for this before I do any conversions on my end, as well as tracing back the reason for this if any in particular and finding out how to point to these specific genes in a secure way. Thank you!
